docs/site/content/repo-docs/reference/configuration.mdx

```diff
@@ -8,7 +8,10 @@ import { InVersion } from '#/components/in-version';
 import { ExperimentalBadge } from '#/components/experimental-badge';
 import Link from 'next/link';
 
-Configure the behavior of `turbo` by adding a `turbo.json` file in your Workspace's root directory.
+Configure the behavior of `turbo` by using a `turbo.json` file in your Workspace's root directory. You can also:
+
+- Use [Package Configurations](/repo/docs/reference/package-configurations) for more granular control.
+- Use `turbo.jsonc` to add comments to your configuration.
```

Comment:
This almost makes it sound like `turbo.json` doesn't allow comments. Maybe frame it as editor support for comments in `turbo.json`?

Reply:
Fair.